app/controllers/api/subcollections/alerts.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  module Subcollections
+    module Alerts
+      def alerts_query_resource(object)
+        object.try(:miq_alert_statuses).to_a
```

Comment:
This is so providers with no miq_alert_statuses associated do not raise an error:
```undefined method `each' for nil:NilClass```